ckanext/data_qld_theme/templates/scheming/form_snippets/resource_visibility.html

```diff
@@ -0,0 +1,54 @@
+{% import 'macros/form.html' as form %}
+{% import 'macros/data_qld_form.html' as data_qld_form %}
+
+
+{%- set field_options = h.data_qld_get_select_field_options(field.field_name) -%}
+{%- set pkg_dict = h.data_qld_get_package_dict(data['package_id']) -%}
+{%- set required = pkg_dict.de_identified_data == 'YES' -%}
+
+{%- set options=[] -%}
+{%- set form_restrict_choices_to=field.get('form_restrict_choices_to') -%}
+{%- if not h.scheming_field_required(field) or
+    field.get('form_include_blank_choice', false) -%}
+  {%- do options.append({'value': '', 'text': '', 'attrs': {}}) -%}
+{%- endif -%}
+{%- for c in h.scheming_field_choices(field) -%}
```

Comment:
can you use 'choice' or similar  instad c  for the variable name?

Reply:
refactored